src/main/java/com/epam/idea/core/social/util/SecurityUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.idea.core.social.util;
+import com.epam.idea.core.model.CommonUserDetails;
+import com.epam.idea.core.model.User;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class SecurityUtil {
+
+	public static void logInUser(User user) {
+		CommonUserDetails userDetails = CommonUserDetails.getBuilder()
```

Comment:
May be, it makes sense to have method for simple case, that takes only user and creates CommonUserDetails